core/settings.py

```diff
@@ -77,6 +75,7 @@ def _generate_settings(self) -> None:
         self.password = self._get_password()
         self.zip_code = self._get_zip_code()
         self.languages = self._get_languages()
+        self.categories = self._get_categories()
```

Comment:
There is no `self.catgories = []` in `def __init__`
So this will only work on first execution but not on the next run where it reads from file

Reply:
You need to add  the category settings loaded from file into `self.categories` variable in `def _load_user_settings`

Reply:
You need to save `self.categories` into the settings file in `def _save_settings`

Reply:
I have done that, can you check once if it is the changes you wanted?